tool/tsh/tsh.go

```diff
@@ -1985,10 +1985,10 @@ func listNodesAllClusters(cf *CLIConf) error {
 		var listings nodeListings
 		for {
```

Comment:
Isn't it better to just let each goroutine accumulate directly into `listings` if we protect the append with a mutex?
It would probably take less time than using two channels + context done to understand when it terminates the group call.

Reply:
Yep, I was just fixing current bug, but agree, that we probably make it simpler with just mutex. I can change it.
@rosstimothy what do you think?

Reply:
I've changed it to use mutex, on the dbs listing as well.